cranelift/codegen/meta/src/gen_listing.rs

```diff
@@ -0,0 +1,171 @@
+//! Generate human-readable lists of instructions and their encodings.
+//!
+//! To display the generated contents, one could use
+//! `find target -name 'listing-x86*' | xargs cat | less` from the top-level project directory.
```

Comment:
Could you please add to this comment, info on what command to run to cause the listing files to be generated in the first place?  Iow, make this as easy as possible for the uninitiated to use :)

Reply:
I actually removed this commit and @bnjbvr is reviewing it separately in https://github.com/bytecodealliance/wasmtime/pull/1191.